src/lib/resources/record.resource.ts

```diff
@@ -85,4 +85,37 @@ export class RecordResource extends BaseResource {
     const response = await this.api.record.registerRecord(record, s3url);
     return this.getVoFromResponse<RecordResponse>(response, 'RecordVO');
   }
+
+  public async registerRecordAndAddStorage(record: RecordVO, s3url: string) {
+    const response = await this.api.record.registerRecord(record, s3url);
+    if (!response.isSuccessful) {
+      throw new PermSdkError(
+        'record could not be registered',
+        response.Results[0].message
+      );
+    }
+    const returnedRecord = this.getVoFromResponse<RecordResponse>(response, 'RecordVO');
+    console.log(returnedRecord);
+    const accountResponse = await this.api.account.getSessionAccount();
+    if (!accountResponse.isSuccessful) {
+      throw new PermSdkError(
+        'could not get the session account',
+        response.Results[0].message
+      );
+    }
+    const account = accountResponse.Results[0].data[0].AccountVO;
+    // should i use returnedRecord here, or count on the client to send us the size correctly?
```

Comment:
Probably `returnedRecord` - I know this is path is for a trusted client, but we should still take a verify approach.

Reply:
Ah thanks for actually seeing this and the quick response :)